static/css/misc.less

```diff
@@ -1030,24 +1030,23 @@
     - cfgov-misc
 */
 
+
+// Size of cicular background.
+@size: 1.5em;
```

Comment:
Yeah I wondered if we wanted to discourage changing the units though. CF has a mix of both https://github.com/cfpb/cf-core/blob/gh-pages/src/cf-vars.less#L49-L53